modules/order/src/Tests/OrderEmptyStoreTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
```

Comment:
We no longer have @file docblocks in Commerce, Drupal has removed them from the coding standard.